docs/community/translate.md

```diff
@@ -7,11 +7,39 @@ keywords:
     - Community
     - Subspace
     - Translate
+    - Localization
 ---
+import TranslationLeaderboard from '@site/src/components/TranslationLeaderboard/index.jsx';
 
 # Translation Guide
 
-This guide will show you how to provide translations for this documentation.
+Welcome to the Subspace Docs translation guide! This guide is here to help you contribute to our goal of making Subspace more accessible and inclusive by providing translations. Subspace Labs is driven by a vision of a decentralized and equitable future, and we believe that overcoming language barriers is crucial to achieving this vision.
+
+## Mission and Vision
+
+Subspace Labs is inherently driven by the vision of a more equitable and decentralized future. We believe that to truly fulfill our vision, we need content that caters to the linguistic diversity of the global community. Thus, the Subspace Labs Translation Initiative is born.
+
+### Our mission
+1. Deliver translated versions of our content, empowering visitors worldwide to learn about Subspace Labs in their language.
```

Comment:
Here as well, Labs or Network?

Reply:
Thanks for catching, got it adjusted to Network 🙏